docs/intro.rst

```diff
@@ -1,8 +1,165 @@
 Introduction
 ============
 
-.. toctree::
+CumulusCI helps development teams build great applications on the Salesforce platform by automating org setup, testing, and deployment.
 
-   learn_more
-   faq
-   read_next
+
+
+Automation with CumulusCI
+-------------------------
+
+If your product development lifecycle and release process is anything like ours at Salesforce.org, it's complex. You're managing multiple packages, dependencies, orgs, and release versions. Not to mention managing org metadata and all the setup operations that need to run in the right sequence, before or after a package is installed, to create a properly configured org. 
+
+Consider the example of Nonprofit Success Pack (NPSP), one of Salesforce.org's flagship open source products. NPSP is a large, complex, heterogeneous application. It consists of six managed packages, with complex dependency relationships. Using automation, all five dependent packages are deployed in the right sequence, the unpackaged record types for the Account and Opportunity objects are delivered, and the final configurations to make the customer's experience better, such as setting up Global Actions and delivering translations, are performed. Biweekly NPSP releases are released to tens of thousands of customer orgs -- with the proper configurations, and without requiring end users to work through a lengthy setup guide.
+
+The CumulusCI suite of tools is part of the not-so-secret sauce that makes it possible to build and release products. CumulusCI automation runs throughout our development lifecycle, starting from feature branches through the delivery of the latest release. 
+
+* GitHub repositories serve as version-controlled source repositories.
+* The CumulusCI command-line interface, ``cci``, runs single-action tasks and multiple-action flows for development and testing.
+* MetaCI uses CumulusCI flows to build Salesforce managed packages from GitHub repositories.
+* MetaDeploy automates setup and configuration of customer orgs.
+
+You can use the very same automation used internally by Salesforce to quickly:
+
+* Build sophisticated orgs with dependencies automatically installed.
+* Load and capture sample datasets to make your orgs feel real.
```

Comment:
"feel real"? Do we have a better suggestion for this one?

Reply:
"feel more like a production environment."